services/inventory/services.go

```diff
@@ -19,24 +19,27 @@ package inventory
 import (
 	"context"
 
-	"github.com/percona/pmm/api/inventorypb"
+	"github.com/AlekSi/pointer"
 	"gopkg.in/reform.v1"
 
 	"github.com/percona/pmm-managed/models"
 	"github.com/percona/pmm-managed/services"
+	"github.com/percona/pmm/api/inventorypb"
```

Comment:
Please follow our guidelines regarding imports.
1. go libs
2. 3rd party libs
3. packages from the same repository

Reply:
fixed